cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -337,9 +337,9 @@ public CoachMarkBuilder setOnDismissListener(OnDismissListener listener) {
          * Set which animation will be used when displaying/hiding the coach mark
          * 
          * @param animationStyle
-         *      the resource ID of the animation to be shown
+         *      the resource ID of the Style to be used for showing and hiding the coach mark
          */
-        public CoachMarkBuilder setAnimation(@AnimRes int animationStyle) {
+        public CoachMarkBuilder setAnimation(@StyleRes int animationStyle) {
```

Comment:
no but passing @AnimRes is a bug - no animations will be shown. I can relax this to int, but then it's really not helping people from falling into doing the same problem